pr_testing/test_multiple_prs.sh

```diff
@@ -786,6 +787,30 @@ if ! $CMSDIST_ONLY ; then # If a CMSSW specific PR was specified #
     exit 0
   fi
 
+  if $PRODUCTION_RELEASE ; then
+    pushd ..
+      mv src src.tmp
+      mkdir src && cd src
+      DSIZE=0
+      THRDS=""
+      NUM_PROC=$(nproc)
+      if [ $NUM_PROC = "0" ] ; then NUM_PROC=1; fi
+      if $USE_IB_TAG ; then git cms-init --upstream-only $CMSSW_IB ; else git cms-init --upstream-only ; fi
```

Comment:
@iarspider , replace this with `git cms-init --upstream-only && git checkout -b codechecks $CMSSW_IB` ( this is what run-pr-code-checks script was doing)